cdap-explore/src/main/java/co/cask/cdap/explore/service/hive/BaseHiveExploreService.java

```diff
@@ -1149,6 +1186,16 @@ protected Object tColumnToObject(TColumnValue tColumnValue) throws ExploreExcept
     throw new ExploreException("Unknown column value encountered: " + tColumnValue);
   }
 
+  private String getNamespaceResolvedSchedulerQueue(Id.Namespace namespaceId) {
```

Comment:
This method is duplicated in 3 places. We should probably re-factor this logic out into a class.